lightrag/llm.py

```diff
@@ -69,12 +69,15 @@ async def openai_complete_if_cache(
     response = await openai_async_client.chat.completions.create(
         model=model, messages=messages, **kwargs
     )
-
+    content = response.choices[0].message.content
+    if r"\u" in content:
+        content = content.encode("utf-8").decode("unicode_escape")
+    print(content)
```

Comment:
Should the `print` statement here be removed?

Reply:
It's for debug, should be removed. I have fixed follow your advice.